src/main/resources/templates/db_scripts/budget_encumbrances_rollover.sql

```diff
@@ -211,6 +219,11 @@ CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.rollover_order(_order_id
                     LEFT JOIN ${myuniversity}_${mymodule}.fund fund ON fund.id = tr.fromFundId
                     WHERE fund.ledgerId::text=_rollover_record->>'ledgerId' AND tr.fiscalYearId::text = _rollover_record->>'fromFiscalYearId'
                           AND tr.jsonb->'encumbrance'->>'sourcePurchaseOrderId'=_order_id;
+        END IF;
+
+        -- #6.1 stop order processing for rolloverType != Preview. If rolloverType = Preview then proceed with validations and result building taking into account only requested ledgerId.
+        IF _rollover_record->>'rolloverType' <> 'Preview' AND array_length(related_not_rollovered_ledger_ids, 1) > 0
```

Comment:
I think  this is incorrect as we still need to validate transaction that is for selected ledger, additionally if rollover is commit and there are no cross-ledger transactions, it will skip validation